src/inmanta/plugins.py

```diff
@@ -21,40 +21,35 @@
 import inspect
 import os
 import subprocess
+import typing
 import warnings
-from collections import abc
-from typing import TYPE_CHECKING, Any, Callable, Literal, Optional, Type, TypeVar
 
+import inmanta.ast as ast
```

Comment:
I like this to keep them short (i.e. snip off as much as possible)

Reply:
I like this too